js/backup.js

```diff
@@ -769,10 +792,15 @@
   }
 
   Whisper.Backup = {
-    backupToDirectory: function() {
+    backupToDirectory: function(options) {
       return getDirectory().then(function(directoryEntry) {
         var idb;
         var dir;
+        conversations = 0;
+        failedConversations = 0;
+        attachments = 0;
+        failedAttachments = 0;
```

Comment:
Minor: To distinguish counters from domain objects, e.g. `conversations: Array<Conversation>`, `failedAttachments: Array<Attachment>`, in the past we would prefix them with `num*`, e.g. `numConversations: number`, `numFailedAttachments: number `, etc.